### ISSUE TITLE
VTK not found for Python 3.7.3

### ISSUE THREAD

**Issue by cbedetti** [scilus/scilpy#45]
`pip install -r requirements.txt` is not working under Python 3.7.3 (fresh venv)

```
Collecting vtk==8.1.0 (from -r requirements.txt (line 18))
  Could not find a version that satisfies the requirement vtk==8.1.0 (from -r requirements.txt (line 18)) (from versions: 8.1.2)
No matching distribution found for vtk==8.1.0 (from -r requirements.txt (line 18))
```

**Comment by GuillaumeTh** [comment]
Hi @cbedetti.
That's because Python 3.7 support only vtk 8.1.2. We didn't test the scripts with this new vtk version.

**Comment by cbedetti** [comment]
No problem
Just want to let you know :+1:

**Comment by jchoude** [comment]
@GuillaumeTh can we either update to use 8.1.2, or change the requirements to >=8.1.0? This way it would also work for those versions of Python.

This would probably not change anything for reproducibility, since we don't test those things for visu right now.

**Comment by GuillaumeTh** [comment]
@jchoude Yes, I will update the requirements.

**Comment by jchoude** [comment]
So, PR #74 was closed without resolution. I know that we mainly pull this dependency from Fury. How do you want to handle it @GuillaumeTh ?

**Comment by arnaudbore** [comment]
Has been fixed in #168
